app.py

```diff
@@ -83,6 +83,16 @@ def category_or_char(char):
             (char,),
         )
 
+    # Redirect single characters to their names
+    if len(char) == 1:
```

Comment:
Because of grapheme clusters like flags we'd want a grapheme-aware comparison here. Maybe we need to add another library? Maybe https://pypi.org/project/grapheme?

Reply:
What would you do in that case? 🤔 Show a page listing all characters pasted and linking to each page?

That feels like a separate feature to me, mostly because I'm pretty much always looking up single-character characters. I like the idea of also handling graphemes though.